### PR TITLE
Update README and add SMART Health IT section

### PR DESCRIPTION
**Pull request by kolkheang**
- Update README
- Remove reference to `./deploy.sh` in tutorial
- Add section on interoperability with SMART Health IT Sandbox

@kpshek 
@mjhenkes 
@zplata

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -444,27 +423,54 @@ window.drawVisualization = function(p) {
...
-$ ./deploy.sh
-```
+Now that we have a snazzy SMART app, it's time to test it.
+
+Next log back into the <a href="https://code.cerner.com/developer/smart-on-fhir/apps" target="_blank">code console</a> and click on the app you've registered (My amazing SMART app). To launch your app through the code console click the "Begin Testing" button. The console will ask if the app you're launching requires a patient in context. Our app requires a patient, so select yes and choose a patient. Please note the millennium username and password, you'll need this credential when prompted. Finally, click launch and the console will redirect to your application.
+
+
+# Run your app against SMART Health IT Sandbox
+
+One of the reasons why SMART on FHIR app is amazing is because of the interoperability factor!  If an EHR follows the SMART and FHIR specifications, your application will work with that EHR's SMART on FHIR implemenmtation.  Let's see if the app that you've built will work with <a href="https://sandbox.smarthealthit.org" target="_blank">SMART Health IT Sandbox</a>.  The following steps will walk you through setting up your app at SMART Health IT Sandbox.
```
Comment (zplata) [inline source/index.html.md]: nit: why SMART on FHIR is amazing

File: source/index.html.md [hunk 2]
```diff
@@ -444,27 +423,54 @@ window.drawVisualization = function(p) {
...
 
-> Or make sure your gh-pages branch is up to date.
+* Go to <a href="https://sandbox.smarthealthit.org" target="_blank">https://sandbox.smarthealthit.org</a> (create an account or sign in)
+* Under My Sandboxes, select SMART DSTU2 Sandbox
+* You will come to a list of your Registered Sandbox Apps. There are 3 SMART apps automatically configured on your account (BP Centiles, Cardiac Risk, and Growth Chart). Add your SMART app tutorial by clicking “Register Manually” under the “Registered Sandbox Apps” title
```
Comment (zplata) [inline source/index.html.md]: * Add your SMART example app

File: source/index.html.md (hunk 1 above)
Comment (kolkheang) [inline source/index.html.md]: :) I'll find another adjective

Comment (zplata) [inline source/index.html.md]: I was talking about just taking out the word app, and leaving the concept name :)

**Pull request merged**